Packs/GitHubITAdmin/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.yml

```diff
@@ -0,0 +1,268 @@
+category: IT Services
+commonfields:
+  id: GitHub IT Admin
+  version: -1
+configuration:
+- defaultvalue: ""
+  display: GitHub URL https://<domain>.github.com/
+  name: url
+  required: true
+  type: 0
+- defaultvalue: ""
+  display: token
+  name: token
+  required: true
+  type: 4
+- defaultvalue: ""
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- defaultvalue: ""
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: ""
+  display: organization name
+  name: org
+  required: false
+  type: 0
+- additionalinfo: An optional custom mapping that takes custom values in the SCIM
+    data into the integration.
+  defaultvalue: ""
+  display: Custom Mapping for Create User
+  name: customMappingCreateUser
+  required: false
+  type: 12
+- additionalinfo: An optional custom mapping that takes custom values in the SCIM
+    data into the integration.
+  defaultvalue: ""
+  display: Custom Mapping for UpdateUser
+  name: customMappingUpdateUser
+  required: false
+  type: 12
+description: GitHub Integration consists of a set of API endpoints that allow customers
+  to automate provisioning of GitHub organization membership.
+display: GitHub IT Admin
+name: GitHub IT Admin
+script:
+  commands:
+  - arguments:
+    - description: SCIM content in JSON format
+      name: scim
+      required: true
+    description: Get a user detail
+    name: get-user
+    outputs:
+    - contextPath: GetUser
+      description: Command context path
+      type: unknown
+    - contextPath: GetUser.active
+      description: Gives the active status of user. Can be true of false.
+      type: boolean
+    - contextPath: GetUser.brand
+      description: Name of the Integration
+      type: string
+    - contextPath: GetUser.details
+      description: Gives the user Profile information if the API is success else error
+        message
+      type: string
+    - contextPath: GetUser.email
+      description: Value of email ID passed as argument
+      type: string
+    - contextPath: GetUser.errorCode
+      description: HTTP error response code
+      type: number
+    - contextPath: GetUser.errorMessage
+      description: Reason why the API is failed
+      type: string
+    - contextPath: GetUser.id
+      description: Value of id passed as argument
+      type: string
+    - contextPath: GetUser.instanceName
+      description: Name of the instance used for testing
+      type: string
+    - contextPath: GetUser.success
+      description: Status of the result. Can be true or false.
+      type: boolean
+    - contextPath: GetUser.userName
+      description: Value of username passed as argument
+      type: string
+  - arguments:
+    - description: SCIM content in JSON format
+      name: scim
+      required: true
+    - description: An optional custom mapping that takes custom values in the SCIM
+        data into the integration.
+      name: customMapping
```

Comment:
```suggestion
      name: custom_mapping
```

Reply:
this is a part of design and consistent in all the integrations for User Provisioning